vg/vg.go

```diff
@@ -17,6 +17,8 @@ import (
 // graphics.
 // The origin is in the bottom left corner.
 type Canvas interface {
+	DPI() float64
+
```

Comment:
Why did we need to add the `DPI` method?

Reply:
for the text.Latex handler.